tools/base/requirements.txt

```diff
@@ -428,6 +430,10 @@ imagesize==1.2.0 \
     --hash=sha256:6965f19a6a2039c7d48bca7dba2473069ff854c36ae6f19d2cde309d998228a1 \
     --hash=sha256:b1f6b5a4eab1f73479a50fb79fcf729514a900c341d8503d62a62dbc4127a2b1
     # via sphinx
+importlib-metadata==4.11.3 \
```

Comment:
this shouldnt be added - its only needed python<3.10 - can you recompile using python3.10 (in a container or similar should work)